src/apps/dashboard/routes/scheduledtasks/index.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from 'react';
+import Page from 'components/Page';
+import globalize from 'lib/globalize';
+import Box from '@mui/material/Box';
+import Stack from '@mui/material/Stack';
+import { useTasks } from '../../features/scheduledtasks/api/useTasks';
+import { getCategories, getTasksByCategory } from '../../features/scheduledtasks/utils/tasks';
+import Loading from 'components/loading/LoadingComponent';
+import Tasks from '../../features/scheduledtasks/components/Tasks';
+import type { TaskInfo } from '@jellyfin/sdk/lib/generated-client/models/task-info';
+import serverNotifications from 'scripts/serverNotifications';
+import Events, { Event } from 'utils/events';
+import ServerConnections from 'components/ServerConnections';
+import { ApiClient } from 'jellyfin-apiclient';
+
+const ScheduledTasks = () => {
+    const { data: initialTasks, isLoading } = useTasks({ isHidden: false });
+    const [tasks, setTasks] = useState<TaskInfo[] | null>(null);
+
+    // TODO: Replace usage of the legacy apiclient when websocket support is added to the TS SDK.
+    useEffect(() => {
+        const apiClient = ServerConnections.currentApiClient();
```

Comment:
The `useApi` hook should be used to get the legacy apiclient too 😉

https://github.com/jellyfin/jellyfin-web/blob/master/src/hooks/useApi.tsx#L11